scripts/generatePortVersionsDb.py

```diff
@@ -0,0 +1,76 @@
+import os
+import os.path
+import sys
+import subprocess
+import json
+import time
+
+from subprocess import CalledProcessError
+from json.decoder import JSONDecodeError
+from pathlib import Path
+

```

Comment:
```suggestion

SCRIPT_DIRECTORY = path.dirname(path.abspath(__file__))

```

allows us to use file-relative paths later, rather than CWD-relative.